ncm-dpmlfc/src/main/pan/components/dpmlfc/schema.pan

```diff
@@ -125,11 +147,53 @@ type ${project.artifactId}_component_dpns_node_config = {
         "readonly" ? boolean
 };
 
+# DAV related parameters: see zlcgdm-dav.conf for parameter documentation
+type ${project.artifactId}_component_dav_node_config = {
+	"DiskAnonUser" ? string
+	"DiskFlags" ? string[]
+	"NSAnonUser" ? string
+	"NSFlags" ? string[]
+	"NSMaxReplicas" ? long
+        "NSRedirectPort" ? long[] with length(SELF) == 2
+	"NSSecureRedirect" ? string with match(SELF,'on|off')
```

Comment:
The problem is that a boolean will not be translated as on/off by the rule-based editor (it will translate it into yes/no if I'm right). Thus the use of a string.